### ISSUE TITLE
Learning Rate Decay in the CLI version

### ISSUE THREAD

**Issue by ogru** [microsoft/LightGBM#2527]
I am running a CLI version and I am trying to provide a list of learning rates to LightGBM but it reports an error when I do something like this:

learning_rate = 0.1,0.05,0.04,0.01

Is there any/better way to do this?

Many thanks!

**Comment by guolinke** [comment]
Learning rate schedule is not supported in CLI version.
